### PR TITLE
remove useless code and fix typo error

### PR DESCRIPTION
**Pull request by googs1025**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
fix some typo error 
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/synchromanager/clustersynchro_manager.go [hunk 1]
```diff
@@ -604,12 +604,12 @@ func buildClusterConfig(cluster *clusterv1alpha2.PediaCluster) (*rest.Config, er
 	}
 
 	if cluster.Spec.APIServer == "" {
-		return nil, errors.New("Cluster APIServer Endpoint is required")
+		return nil, errors.New("cluster APIServer Endpoint is required")
```
Comment (Iceber) [inline pkg/synchromanager/clustersynchro_manager.go:607]: Here they are treated as a special whole that can begin with a capitalization.

File: pkg/synchromanager/clustersynchro_manager.go [hunk 2]
```diff
@@ -604,12 +604,12 @@ func buildClusterConfig(cluster *clusterv1alpha2.PediaCluster) (*rest.Config, er
...
 	}
 
 	if len(cluster.Spec.TokenData) == 0 &&
 		(len(cluster.Spec.CertData) == 0 || len(cluster.Spec.KeyData) == 0) {
-		return nil, errors.New("Cluster APIServer's Token or Cert is required")
+		return nil, errors.New("cluster APIServer's Token or Cert is required")
```
Comment (Iceber) [inline pkg/synchromanager/clustersynchro_manager.go:612]: ditto

**Review by Iceber — COMMENTED** [review]
Some changes needed, others look good!

Thanks for your contribution.

File: pkg/synchromanager/clustersynchro_manager.go (hunk 1 above)
Comment (googs1025) [reply pkg/synchromanager/clustersynchro_manager.go:607]: ok, I got it, thank you for your review.

Comment (googs1025) [reply pkg/synchromanager/clustersynchro_manager.go]: @Iceber  Already modified, thanks for reviewing again.

**Review by Iceber — APPROVED** [review]
LGTM, Thanks

**Pull request merged**